docs/quickstart/docker.md

```diff
@@ -90,12 +97,14 @@ ENV SPARK_VERSION=2.3.3
 ENV HADOOP_VERSION=2.7
 
 # support Kerberos certification
-RUN install -yq curl unzip wget grep sed vim krb5-user libpam-krb5 && apt-get clean
+RUN export DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get install -yq krb5-user libpam-krb5 && apt-get clean
```

Comment:
Why this change ？It seems not related with timezone

Reply:
When installing `krb5` in the dockerfile, Need to set `DEBIAN_FRONTEND=noninteractive`, Otherwise it will wait for user input.

This docker.md file is a documentation for zeppelin interpreter on docker, so it was updated.

Reply:
it could help to explain that in comment